frontend/pages/policies/ManagePoliciesPage/components/CalendarEventsModal/_styles.scss

```diff
@@ -36,10 +36,30 @@
   .form-field__help-text {
     margin-top: $pad-large;
   }
+
+  &__hide-main {
```

Comment:
Nice. I think you'll need to hide the app-wide overlay from the original modal as well, or something
like that.

Reply:
Done.